src/rules/property-no-unknown/index.js

```diff
@@ -37,6 +38,7 @@ export default function (actual, options) {
       if (!shouldCheckPrefixed && vendor.prefix(prop)) { return }
 
       if (!isStandardSyntaxProperty(prop)) { return }
+      if (!isStandardSyntaxDeclaration(decl)) { return }
       if (isCustomProperty(prop)) { return }
```

Comment:
I think we should move up `isStandard*` and `isCustomProperty` than `if (!shouldCheckPrefixed && vendor.prefix(prop)) { return }`. Why check prefixes on non standard properties? It is not you small mistake, but we can do this in this PR.